website/docs/components/app-header/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: App Header
+description: A header navigation that provides access to global elements and application utilities.
+caption: Used as a global and utility navigation within an application.
+links:
+  figma: {link to the "AppHeader" page in the components Figma library}
```

Comment:
Only blocking this so that we remember to add these in (the Figma branch has yet to be approved, so I can't add a link here yet).

Reply:
Are these ready yet?

Reply:
Yep, if you're working on the docs, can you add this? https://www.figma.com/design/noyY6dUMDYjmySpHcMjhkN/HDS-Components?node-id=58460-1572

Reply:
Does this look ok?

https://github.com/hashicorp/design-system/pull/2239/commits/e6e9e47d76707b03e5412c6b22ff221c0df4c1b7